lib/engine/corporation.rb

```diff
@@ -15,7 +15,7 @@ class Corporation
     include Spender
 
     attr_accessor :ipoed, :par_price, :share_price, :tokens
-    attr_reader :companies, :coordinates, :min_price, :name, :full_name, :logo, :trains, :color
+    attr_reader :companies, :coordinates, :min_price, :name, :full_name, :logo, :trains, :color, :or_revenue_history
```

Comment:
rename to revenue_history

Reply:
will do. renaming things -- the bane of dynamically typed languages :)